keysniffer.c

```diff
@@ -107,95 +108,102 @@ static struct notifier_block keysniffer_blk = {
 	.notifier_call = keysniffer_cb,
 };
 
-void to_base_n_string(int input, int base,char *rep,int size){
-  int i;
-  /* buffer to hold the remainders in reverse order */
-  int div = input;  
-  int remainders[size];
-  int start;
-  /* zero out the buffer */
-  for ( i = 0; i < size; i++)
-    rep[i] = 0;
-
-  if( !input ){
-    rep[0] = '0';
-    return;
-  }
-
-  /* input the remainders in reverse order */
-  i = size - 1; 
-  while( div > 0 ){
-    remainders[i] = div % base;
-    div = div / base;
-    i--;
-  }
-  start = i+1;
-  /* place them into the string starting from the beginning 
-     with the proper character code */
-  for( i = start; i < size; i++){
-    rep[i - start] = '0' + remainders[i];
-  }
-      
+/*
+ * Function to convert an integer to a base n string representation
+ *
+ * Parameters :
+ *  - input : integer to be converted
+ *  - n     : numerical base
+ *  - rep   : string buffer to place the representation in
+ *  - size  : the max size of the buffer
+ */
+void to_base_n_string(int input, int n,char *rep,int size){
```

Comment:
Please add spaces between `int n,char *rep,int size)`
The `{` should be on the next line.